examples/net_framework/README.md

```diff
@@ -0,0 +1,172 @@
+# Building DeepSpeech for Windows
+
+Now we can build the native DeepSpeech library and run inference on Windows using the C# client.
+
+**Table of Contents**
+
+- [Prerequisites](#prerequisites)
+- [Getting the code](#getting-the-code)
+- [Configuring the paths](#configuring-the-paths)
+- [Adding environment variables](#adding-environment-variables)
+    - [MSYS2 paths](#msys2-paths)
+    - [BAZEL path](#bazel-path)
+    - [Python path](#python-path)
+    - [CUDA paths](#cuda-paths)
+- [Building the code](#building-the-code)
+    - [Build for CPU](#cpu)
+    - [Build with CUDA support](#gpu-with-cuda)
+- [Using the generated library](#using-the-generated-library)
+## Prerequisites
+
+* [Python 3.6](https://www.python.org/)
+* [Git Large File Storage](https://git-lfs.github.com/)
+* [MSYS2](https://www.msys2.org/)
+* [Bazel v0.17.2](https://github.com/bazelbuild/bazel/releases)
+* [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)
+* Windows 10
+* [Visual Studio 2017 Community](https://visualstudio.microsoft.com/vs/community/) 
+
+Inside the Visual Studio Installer enable `MS Build Tools` and `VC++ 2015.3 v14.00 (v140) toolset for desktop`
```

Comment:
Are those hard requirements or we can expect it works with other versions ?

Reply:
I've tried a lot o versions and can say these requirements are not optional if you want to build for GPU. In my latest test I was able to build for CPU using the latest versions of VS Build Tools, but it failed for GPU. For CPU only it may work with other versions.